content/events/index.html.haml

```diff
@@ -137,10 +137,11 @@ notitle: true
       + '&amp;ctz='
 
     var calendarSrcPrefix = '<iframe src="' + calendarSrc;
-    var calendarSrcSuffix = '" style=" border-width:0 " width="1024" height="600" frameborder="0" scrolling="no"></iframe>';
+    
```

Comment:
why move the `calendarSrcSuffix` into the function? Please either move it back or remove this extra blank line.